src/applications/vaos/components/ExpressCareReasonField.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import classNames from 'classnames';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { Link } from 'react-router';
+
+export default function ExpressCareReasonField({
+  uiSchema,
+  formData,
+  onChange,
+  errorSchema,
+  formContext,
+  onError,
+}) {
+  const { reason, additionalInformation } = formData;
+  const hasError =
+    formContext.submitted && errorSchema.reason.__errors.length > 0;
+
+  const radioGroupClasses = classNames({
+    'usa-input-error': hasError,
+  });
+
+  return (
+    <>
+      <h2>Tell us about your health concern</h2>
+      <fieldset>
+        <legend
+          id="vaos-express-care__radiogroup-label"
+          className="vads-u-font-size--base vads-u-color--gray-dark vads-u-font-weight--normal"
+        >
+          Please select a concern from the list the best fits your needs today.
+          <span className="schemaform-required-span">(*Required)</span>
+        </legend>
+        <div
+          className={radioGroupClasses}
+          role="radiogroup"
+          aria-labelledby="vaos-express-care__radiogroup-label"
+        >
+          {hasError && (
+            <span className="usa-input-error-message" role="alert">
+              <span className="sr-only">Error</span> Please select a symptom
+            </span>
+          )}
+          {uiSchema.options.items.map((option, index) => {
+            const checked = option.value === reason;
+            const divClasses = classNames('vads-u-padding-left--1', {
+              'vads-u-padding-bottom--1': checked,
+              'vads-u-border-color--primary-alt-light': checked,
+              'vads-u-border-color--white': !checked,
+              'vads-u-border-left--7px': !hasError,
+              'vads-u-margin-left--neg2': !hasError,
+            });
+
+            return (
+              <div key={`reason-radio-${index}`} className={divClasses}>
+                <input
+                  type="radio"
+                  id={`vaos-express-care__radio-${option.id}`}
+                  name="vaos-express-care__radio"
```

Comment:
The form system name would look like this: `root_reasonForRequest_reason` (it would also use this as the `id`, but with the index appended, too), I think we could use that to keep sort of inline.